webpack.test.config.js

```diff
@@ -1,65 +1,12 @@
-var webpack = require( 'webpack' ),
+const webpack = require( 'webpack' ),
 	path = require( 'path' ),
-	nodeExternals = require( 'webpack-node-externals' );
+	nodeExternals = require( 'webpack-node-externals' )
```

Comment:
sure. I only used it because it was already like that in the webpack files. I'll change it up.